package.json

```diff
@@ -1,67 +1,72 @@
-    "eslint-plugin-node": "^9.1.0",
-    "eslint-plugin-promise": "^4.1.1",
-    "eslint-plugin-standard": "^4.0.0",
-    "eslint-plugin-unicorn": "^8.0.2",
-    "make-node": "^0.4.6",
-    "mocha": "6.x.x",
-    "typescript": "^3.4.5"
-  },
-  "engines": {
-    "node": ">=8.0.0"
-  },
-  "scripts": {
-    "spec": "mocha --reporter spec --require test/bootstrap/node test/*.test.js test/**/*.test.js",
-    "lint": "eslint --ext js,md . --max-warnings 0",
-    "lintfix": "eslint --ext js,md . --fix",
-    "test": "npm run lint && npm run spec"
-  }
+    "name": "@passport-next/passport",
+    "version": "3.0.0",
+    "description": "Simple, unobtrusive authentication for Node.js.",
+    "author": {
+        "name": "Passport Next Developers",
+        "url": "https://github.com/orgs/passport-next/people"
+    },
+    "homepage": "https://github.com/passport-next/passport",
+    "contributors": [
+        "Brett Zamir"
+    ],
+    "bugs": {
+        "url": "https://github.com/passport-next/passport/issues"
+    },
+    "dependencies": {
+        "@passport-next/passport-strategy": "1.x.x"
+    },
+    "devDependencies": {
+        "@mysticatea/eslint-plugin": "^10.0.3",
+        "@passport-next/eslint-config-passport-next": "1.x.x",
+        "@passport-next/chai-passport-strategy": "1.x.x",
+        "chai": "4.x.x",
+        "chai-connect-middleware": "0.x.x",
+        "eslint": "^5.16.0",
+        "eslint-config-standard": "^12.0.0",
+        "eslint-plugin-compat": "^3.1.1",
+        "eslint-plugin-eslint-comments": "^3.1.1",
+        "eslint-plugin-import": "^2.17.3",
+        "eslint-plugin-jsdoc": "^8.0.0",
+        "eslint-plugin-markdown": "^1.0.0",
+        "eslint-plugin-no-use-extend-native": "^0.4.0",
+        "eslint-plugin-node": "^9.1.0",
+        "eslint-plugin-promise": "^4.1.1",
+        "eslint-plugin-standard": "^4.0.0",
+        "eslint-plugin-unicorn": "^9.1.0",
+        "make-node": "^0.4.6",
+        "mocha": "6.x.x",
+        "nunjucks": "^3.2.0"
+    },
+    "engines": {
+        "node": ">=8.0.0"
+    },
+    "keywords": [
+        "express",
+        "connect",
+        "auth",
+        "authn",
+        "authentication"
+    ],
+    "license": "MIT",
+    "licenses": [
+        {
+            "type": "MIT",
+            "url": "http://opensource.org/licenses/MIT"
+        }
+    ],
+    "main": "./lib",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/passport-next/passport"
+    },
+    "scripts": {
+        "templates": "node ./templates/gen.js",
+        "init-new-project": "node ./templates/gen --init",
+        "lint": "./node_modules/.bin/eslint --report-unused-disable-directives --max-warnings 0 --ext js,md .",
+        "lintfix": "./node_modules/.bin/eslint --ext js,md . --fix",
+        "test": "./node_modules/.bin/mocha --reporter spec --require test/bootstrap/node test/*.test.js test/**/*.test.js",
+        "test-cov": "make test-cov",
```

Comment:
There currently is no `test-cov` in the `Makefile`

Reply:
The make stuff is really old and comes from the upstream repo, I'm working on replacing it, I think the only commands left are regarding test coverage so, I've added a todo for it https://github.com/orgs/passport-next/projects/1#card-22818352